README.md

```diff
@@ -85,26 +85,53 @@ While I am a big fan of Tiled and LdTk, for my case I was looking for something
 
 ## :eyeglasses: Getting started
 
-   ```bash
-   $ git clone https://github.com/blurymind/tilemap-editor.git
-   $ yarn
-   $ yarn start
-   ```
+There are a few ways to get started, depending on how you have things setup. You will need [node](https://nodejs.org/en/) installed.
+
+### local - quick
+
+If you just want to run the editor locally, but aren't going to be developing on it, and just want to run the latest version:
+
+```sh
+npx tilemap-editor@latest
+```
+
+### global - quick
+
+If you'd like to install it, globally:
+
+```sh
+npm i -g tilemap-editor
```

Comment:
that works even now though, right?

Reply:
no. did you try it?

Reply:
here is an example of it not working: https://asciinema.org/a/gPMlSSiDh2lNbbB6ahazONJ9B